content/docs/command-reference/exp/run.md

```diff
@@ -118,6 +118,12 @@ changes between/after queueing runs.
 `dvc exp run --temp`, for example to continue working on the project meanwhile
 (e.g. on another terminal).
 
+> ⚠️ Note that only tracked files and directories will be included in
+> `--queue/temp` experiments. To include untracked files, stage them with
+> `git add -N` first (before `dvc exp run`). Feel free to `git reset` them
```

Comment:
```suggestion
> `git add (-N)` first (before `dvc exp run`). Feel free to `git reset` them
```
Do you mind adding the parentheses back? Otherwise, it seems like `-N` is required.

Reply:
I think it's the recommended way. Much safer than without `-N` for this trick, I think!

This is a temporary patch anyway right? Behavior-wise. It may take a while to decide what to do but still...

Reply:
> Much safer than without `-N` for this trick

Is it? Looking at https://stackoverflow.com/questions/24329051/what-does-git-add-intent-to-add-or-n-do-and-when-should-it-be-used, my understanding is that `-N` is only useful to diff new files and edit their patches line by line. Once a file is staged/tracked by Git, experiments should track all changes to that file regardless of how it was added.

Since `-N` is not a common option, I worry it will confuse people if we recommend it by default. If you think it's not likely to be confusing regardless, I'm fine letting it go.

Reply:
I think it's safer because the file contents don't actually get staged to Git, so even if you forget to `git reset` after, you'll only end up with some empty files in your main Git history instead of potentially unwanted contents. Either way it's slightly a hacky trick, hopefully a temporary patch.

I do agree it's uncommon somay be weary about unknown implications of `-N`... Idk. I'd wait on @pmrowla's opinion on whether to keep `-N` or not, but mentioning both probably adds to confusion.

Reply:
Yeah you know what `-N` seems unnecessary since they'll be committed in the experiment anyway. Merging without it for now but again, we can revisit.

Reply:
Yup, you put it better than I did.